libs/utils/android/workloads/uibench.py

```diff
@@ -85,7 +85,7 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         self.out_dir = out_dir
         self.collect = collect
 
-        # Press Back button to be sure we run the video from the start
+        Screen.unlock(self._target)
         System.menu(self._target)
```

Comment:
A think you can make even more nicely red coloured this patch by removing this and the next call... aren't they already done by the new `Screen.unlock()` call?